mlflow/server/js/src/components/BaggedCell.js

```diff
@@ -24,56 +22,65 @@ export default class BaggedCell extends PureComponent {
     sortIcon: PropTypes.node,
   };
 
-  render() {
-    const { keyName, value, onSortBy, isParam, onRemoveBagged, sortIcon } = this.props;
+  handleSortAscending = () => {
+    const { isParam, keyName, onSortBy } = this.props;
+    const keyType = (isParam ? "params" : "metrics");
+    const canonicalKey = ExperimentViewUtil.makeCanonicalKey(keyType, keyName);
+    onSortBy(canonicalKey, true);
+  };
+
+  handleSortDescending = () => {
+    const { isParam, keyName, onSortBy } = this.props;
     const keyType = (isParam ? "params" : "metrics");
     const canonicalKey = ExperimentViewUtil.makeCanonicalKey(keyType, keyName);
+    onSortBy(canonicalKey, false);
+  };
+
+  handleRemoveBagged = () => {
+    const { isParam, keyName, onRemoveBagged } = this.props;
+    onRemoveBagged(isParam, keyName);
+  };
+
+  render() {
+    const { keyName, value, sortIcon } = this.props;
     const cellClass = classNames("metric-param-content", "metric-param-cell", "BaggedCell");
     return (
       <span
         className={cellClass}
       >
-      <Dropdown id="dropdown-custom-1" style={{width: 250}}>
-        <ExperimentRunsSortToggle
-          bsRole="toggle"
-          className={"metric-param-sort-toggle"}
-        >
-              <span
-                className="run-table-container underline-on-hover"
-                style={styles.metricParamCellContent}
-                title={keyName}
-              >
-                {sortIcon}
-                {keyName}:
-              </span>
-        </ExperimentRunsSortToggle>
-        <span
-          className="metric-param-value run-table-container"
-          style={styles.metricParamCellContent}
+        <Dropdown
+          overlay={(
+            <Menu>
+              <Menu.Item onClick={this.handleSortAscending}>
+                Sort ascending
+              </Menu.Item>
+              <Menu.Item onClick={this.handleSortDescending}>
+                Sort descending
+              </Menu.Item>
+              <Menu.Item onClick={this.handleRemoveBagged}>
+                Display as a separate column
```

Comment:
This line changes `Display in own column` to `Display as a separate column`. @stbof